auto_nag/scripts/assignee_no_login.py

```diff
@@ -64,14 +67,35 @@ def get_mail_to_auto_ni(self, bug):
         if self.has_bot_set_ni(bug):
             return None
 
+        # Avoid to ni if the bug was last touched many years ago and has low priority and low severity.
+        # It's not paramount for triage owners to make an explicit decision here, it's enough for them
+        # to receive the notification about the unassignment from Bugzilla via email.
+        last_change = dateutil.parser.parse(bug["last_change_time"]).replace(
+            tzinfo=None
+        )
+        if (
+            last_change < datetime.datetime.utcnow() - relativedelta(years=3)
```

Comment:
We could move up `datetime.datetime.utcnow() - relativedelta(years=3)` to the constructor.

Reply:
I moved it at the beginning and made it a constant.